bitex/interface/binance.py

```diff
@@ -23,47 +19,39 @@ class Binance(RESTInterface):
 
     # pylint: disable=arguments-differ
 
-
     def __init__(self, **api_kwargs):
         """Initialize class instance."""
         super(Binance, self).__init__('Binance', BinanceREST(**api_kwargs))
 
     def request(self, verb, endpoint, authenticate=False, **req_kwargs):
         """Preprocess request to API."""
         return super(Binance, self).request(verb, endpoint, authenticate=authenticate,
-                                             **req_kwargs)
+                                            **req_kwargs)
 
     def _get_supported_pairs(self):
-        """
-        Generate a list of supported pairs.
-        """
+        """Return a list of supported pairs."""
         r = self.request('GET', 'v1/exchangeInfo').json()
         pairs = [entry['symbol'] for entry in r['symbols']]
         return pairs
 
-    def is_supported(self, pair):
-        return super(Binance, self).is_supported(pair)
+    # Automatically calls the RESTInterface method.
```

Comment:
You can remove this entirely - no need to keep it commented out. It should work fine - this was just a left over from before I made `is_supported` non-abstract.